app/formSchema/uiSchema.ts

```diff
@@ -603,7 +603,7 @@ const uiSchema = {
     'ui:options': {
       maxLength: MAX_LONG_INPUT_LENGTH,
       label: false,
-      fileTypes: '.xlsx',
+      fileTypes: 'xls, .xlsx',
```

Comment:
```suggestion
      fileTypes: '.xls, .xlsx',
```
Should there be a period in front of each of those? 
Additionally, can we create an `EXCEL_FILE_EXTENSIONS` constant in this file to ensure consistency?